### PR TITLE
2824 - adding ability to view build output without authenticating

### PR DESCRIPTION
**Pull request by mgamini**

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-web/app/package-page/PackagePageComponent.ts [hunk 1]
```diff
@@ -155,11 +155,10 @@ export class PackagePageComponent implements OnDestroy {
     }
 
     private fetchBuilds() {
-        if (this.token) {
```
Comment (raskchanky) [inline components/builder-web/app/package-page/PackagePageComponent.ts]: Why remove the check for a valid token, yet still pass it along?

File: components/builder-web/app/packages-page/packages-page.component.ts [hunk 2]
```diff
@@ -188,10 +188,7 @@ export class PackagesPageComponent implements OnInit, OnDestroy {
     private fetchBuilds() {
         if (this.origin && this.name) {
             let token = this.store.getState().gitHub.authToken;
-
-            if (token) {
```
Comment (raskchanky) [inline components/builder-web/app/packages-page/packages-page.component.ts]: Same here as above

File: components/builder-web/app/package-page/PackagePageComponent.ts (hunk 1 above)
Comment (mgamini) [inline components/builder-web/app/package-page/PackagePageComponent.ts]: The API currently accepts a null token. If this is a private package, we'll want that token to be sure the user can access it.

File: components/builder-web/app/packages-page/packages-page.component.ts [hunk 3]
```diff
@@ -188,10 +188,7 @@ export class PackagesPageComponent implements OnInit, OnDestroy {
...
-                this.store.dispatch(fetchBuilds(this.origin, this.name, token));
-            }
+            this.store.dispatch(fetchBuilds(this.origin, this.name, token));
```
Comment (raskchanky) [inline components/builder-web/app/packages-page/packages-page.component.ts]: The server API for showing build jobs doesn't require authentication, which makes me think that passing a token to this function shouldn't need to happen.

Comment (mgamini) [inline components/builder-web/app/packages-page/packages-page.component.ts]: Will that change with private builds?

Comment (raskchanky) [inline components/builder-web/app/packages-page/packages-page.component.ts]: Yeah, good call.  Leave it.

**Comment by raskchanky** [comment]
tenor-182699254

**Comment by raskchanky** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/266082346).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/266082346).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**